xarray/backends/h5netcdf_.py

```diff
@@ -149,9 +150,16 @@ def open(
         decode_vlen_strings=True,
         driver=None,
         driver_kwds=None,
+        storage_options=None,
```

Comment:
Could we get type hints at least here?

Reply:
Yeah, just pushed a commit for that 👍

Reply:
...and fixed things to make `mypy` happy (one day that'll happen on the first try)